sso/sso72-x509-https.json

```diff
@@ -4,12 +4,12 @@
     "metadata": {
         "annotations": {
             "iconClass" : "icon-sso",
-            "tags" : "sso,keycloak,jboss,hidden",
+            "tags" : "sso,keycloak,jboss",
             "version": "1.4.10",
-            "openshift.io/display-name": "Single Sign-On 7.2",
+            "openshift.io/display-name": "Red Hat Single Sign-On 7.2 (Ephemeral with re-encrypt TLS)",
```

Comment:
I'd drop the "with re-encrypt TLS" ... it is the default now, not need to clutter the UI with useless information